tesseract/tesseract_planning/tesseract_motion_planners/src/ompl/ompl_motion_planner.cpp

```diff
@@ -38,6 +39,44 @@ TESSERACT_COMMON_IGNORE_WARNINGS_POP
 
 namespace tesseract_planning
 {
+bool checkStartState(const ompl::base::ProblemDefinitionPtr& prob_def,
+                     const Eigen::Ref<const Eigen::VectorXd>& state,
+                     const OMPLStateExtractor& extractor)
+{
+  if (!(prob_def->getStartStateCount() >= 1))
+    return false;
+
+  for (unsigned i = 0; i < prob_def->getStartStateCount(); ++i)
+    if (extractor(prob_def->getStartState(i)).isApprox(state, 1e-5))
+      return true;
```

Comment:
Is this supposed to be true if any of the states are equal to state or if all are equal?

Reply:
Yea, this is used to check the final trajectory start state and make sure it was one of the provided start states. The same for the end state.